docs/source/infosheet/infosheet.csv

```diff
@@ -5,3 +5,8 @@ Dexcom,Pro CGM,`dexcom/pro_cgm`,OAuth 2.0,yes,link
 Withings,Body+,`withings/bodyplus`,OAuth 2.0,yes,link
 Withings,ScanWatch,`withings/scanwatch`,OAuth 2.0,yes,link
 Polar,Verity Sense,`polar/verity_sense`,only email and password,yes,link
+Cronometer, Cronometer App, 'cromometer/Cronometer', only email and password,yes,link
+MyFitnessPal, MyFitnessPal App, 'myfitnesspal/myfitnesspal', browser cookies, link
+Google, Fitness, 'google/googlefit', authorization_code, link
```

Comment:
include a "yes" for the simulator column in the csv for your devices (i.e. google fit, strava, myfitnesspal)